Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/outbound/deferred/request/StandardOutboundDocSubmissionDeferredRequestTest.java

```diff
@@ -67,61 +70,99 @@
+        StandardOutboundDocSubmissionDeferredRequest dsDeferredReq = createDocSubmissionDeferredRequestOrchImpl();
+        XDRAcknowledgementType response = dsDeferredReq.provideAndRegisterDocumentSetBAsyncRequest(request, assertion,
+            targets, urlInfo);
 
-        context.assertIsSatisfied();
         assertNotNull(response);
         assertEquals(NhincConstants.XDR_ACK_FAILURE_STATUS_MSG, response.getMessage().getStatus());
+        verify(mockAuditLogger).auditRequestMessage(eq(request), eq(assertion), any(NhinTargetSystemType.class),
+            eq(NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION), eq(NhincConstants.AUDIT_LOG_NHIN_INTERFACE),
+            eq(Boolean.TRUE), isNull(Properties.class), eq(NhincConstants.NHINC_XDR_REQUEST_SERVICE_NAME));
     }
 
     @Test
     public void testProvideAndRegisterDocumentSetB_emptyTargets() {
-        expect2MockAudits();
-        XDRAcknowledgementType response = runProvideAndRegisterDocumentSetBAsyncRequest_emptyTargets();
+        NhinTargetCommunitiesType targets = new NhinTargetCommunitiesType();
+        ProvideAndRegisterDocumentSetRequestType request = new ProvideAndRegisterDocumentSetRequestType();
+        final AssertionType assertion = createAssertion(senderHCID);
+        UrlInfoType urlInfo = new UrlInfoType();
+
+        when(mockPolicyChecker.checkXDRRequestPolicy(any(ProvideAndRegisterDocumentSetRequestType.class),
+            any(AssertionType.class), any(String.class), any(String.class),
+            eq(NhincConstants.POLICYENGINE_OUTBOUND_DIRECTION))).thenReturn(Boolean.TRUE);
+
+        when(mockDelegate.process(any(OutboundDocSubmissionDeferredRequestOrchestratable.class))).thenReturn(
+            createOutboundDocSubmissionDeferredRequestOrchestratable(Boolean.TRUE));
+
+        StandardOutboundDocSubmissionDeferredRequest dsDeferredReq = createDocSubmissionDeferredRequestOrchImpl();
+        XDRAcknowledgementType response = dsDeferredReq.provideAndRegisterDocumentSetBAsyncRequest(request, assertion,
+            targets, urlInfo);
 
-        context.assertIsSatisfied();
         assertNotNull(response);
         assertEquals(NhincConstants.XDR_ACK_FAILURE_STATUS_MSG, response.getMessage().getStatus());
+        verify(mockAuditLogger).auditRequestMessage(eq(request), eq(assertion), any(NhinTargetSystemType.class),
+            eq(NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION), eq(NhincConstants.AUDIT_LOG_NHIN_INTERFACE),
+            eq(Boolean.TRUE), isNull(Properties.class), eq(NhincConstants.NHINC_XDR_REQUEST_SERVICE_NAME));
     }
 
     @Test
     public void testHasNhinTargetHomeCommunityId() {
-        StandardOutboundDocSubmissionDeferredRequest entityOrch = createEntityDocSubmissionDeferredRequestOrchImpl();
+        StandardOutboundDocSubmissionDeferredRequest entityOrch = createDocSubmissionDeferredRequestOrchImpl();
 
         boolean hasTargets = entityOrch.hasNhinTargetHomeCommunityId(null);
```

Comment:
The "hasTargets" var is unnecessary, we can pass values directly into assertFalse without losing readability.